local/src/core/YamlParser.cc

```diff
@@ -297,6 +297,56 @@ YamlParser::populate_http_message(YAML::Node const &node, HttpHeader &message)
         } else {
           message._content_size = content_size;
         }
+
+        if (auto rule_node(content_node[YAML_CONTENT_RULE_KEY]); rule_node) {
+          if (rule_node.IsMap()) {
+            // Verification is specified as a map, such as:
+            // rule: { as: equal, case: ignore }
+
+            // Get case setting (default false)
+            auto const rule_case_node{rule_node[YAML_RULE_CASE_MAP_KEY]};
+            bool is_nocase = false;
+            if (rule_case_node && rule_case_node.IsScalar()) {
+              TextView case_str = Localizer::localize(rule_case_node.Scalar());
+              if (case_str == VERIFICATION_DIRECTIVE_IGNORE) {
+                is_nocase = true;
+              }
+            }
+
+            // Get rule type for "as: equal" structure, or "not: equal" if "as" fails
+            TextView rule_type;
+            bool is_inverted = false;
+            if (auto const rule_type_node_as = rule_node[YAML_RULE_TYPE_MAP_KEY]; rule_type_node_as) {
+              rule_type = rule_type_node_as.Scalar();
+            } else if (auto const rule_type_node_not = rule_node[YAML_RULE_TYPE_MAP_KEY_NOT];
+                      rule_type_node_not)
+            {
+              rule_type = rule_type_node_not.Scalar();
+              is_inverted = true;
+            } else if (message._verify_strictly) {
+              rule_type = VERIFICATION_DIRECTIVE_EQUALS;
+            } else {
+              errata.note(
+                  S_INFO,
+                  "Body rule at {} ignored: no directive, and equality is not assumed",
+                  rule_node.Mark());
+              // Cannot use continue statement because of client request/server response
```

Comment:
So this will be emitted when there is a `content` node but not a `rule` directive? If so, the INFO will be pretty noisy, especially in prod sim where most proxy-* content nodes will not have a `rule` directive. I would just make this silent.

Reply:
This should only be emitted when there is a `content` node and there is a `verify` node (after the changes), but the rules specified is bogus.

Reply:
Ouch. Good catch! Please make them S_ERROR too. :) 

I missed this in Will's review (I know I touched those lines last, but only to convert to Alan's new libswoc Errata 
interface). If someone adds a verification rule but doesn't give a directive, we really should make that an ERROR. 

Here's my logic: if someone specifies a `verify` node but forgets to set a `directive`, that should really be a hard stop with an error message. That means they tried to specify verification, but forgot (or didn't understand) that they needed to give a directive. That can only be a mistake and we should make that obvious to them. S_ERROR will provide a non-zero return code to alert the user that they tried to specify a rule but missed specifying a directive.

Reply:
Done.